drivers/i2s/i2s_ll_stm32.c

```diff
@@ -349,9 +349,20 @@ static int i2s_stm32_trigger(const struct device *dev, enum i2s_dir dir,
 			LOG_ERR("DRAIN trigger: invalid state");
 			return -EIO;
 		}
-		stream->stream_disable(stream, dev);
-		stream->queue_drop(stream);
-		stream->state = I2S_STATE_READY;
+		if (dir == I2S_DIR_TX) {
```

Comment:
Should RX case be treated as documented: `	 * If the trigger is applied to the RX queue it has the same effect as * I2S_TRIGGER_STOP.` ?